src/main/kotlin/no/nav/familie/ks/sak/api/FagsakController.kt

```diff
@@ -32,7 +32,6 @@ class FagsakController(private val fagsakService: FagsakService) {
     @PostMapping(path = ["/sok"])
```

Comment:
Skal vi lag separat favro kort for å ta i bruk tilgangservice i kontrollere, eller skal vi kjøre på med det her?

Reply:
Tenker egentlig at vi må ta det som separate oppgaver

Reply:
https://favro.com/organization/98c34fb974ce445eac854de0/077068028bffba85055cca2d?card=Tea-9741

Reply:
Støtter nytt Favrokort!